template/module/README.rst

```diff
@@ -65,8 +65,8 @@ Images
 Contributors
 ------------
 
-* Firstname Lastname <email.address@example.org>
-* Second Person <second.person@example.org>
+* Firstname Lastname <email.address@example.org> (optional company website url)
```

Comment:
I agree with @sbidouil - if I make a contribution an an employee, my employer should be shown alongside my name, since I'm acting on behalf of it.
But @elicoidal also has a point: the company funding the module development deserver some visibility. It would be nice for them to be allowed to display their logos in a section next (below?) to the contributor list.